modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -368,8 +368,8 @@ They should contain the word `should` somewhere in the step text.
 ----
 
 Note that there are often multiple things that `should` or `should not` be the case after the `When` action.
-For example, in the above scenario, various files and folders should have been deleted.
-But other files and folders (that are part of the skeleton) should still be there.
+For example, in the above scenario, various files and folders (that are part of the skeleton) should still be there.
+But other files and folders under `FOLDER` should have been deleted.
```

Comment:
Write these sentences the opposite way around, so this description matches the way that the example scenario above has been written.